script/uplift.py

```diff
@@ -113,14 +113,12 @@ def get_previous_version_branch(version):
 def get_remote_channel_branches(raw_nightly_version):
     global config
     nightly_version = get_current_version_branch(raw_nightly_version)
-    dev_version = get_previous_version_branch(nightly_version)
-    beta_version = get_previous_version_branch(dev_version)
+    beta_version = get_previous_version_branch(nightly_version)
     release_version = get_previous_version_branch(beta_version)
```

Comment:
whoops - this is going to fail... since 1.6 is not there anymore

Reply:
"Fixed" with https://github.com/brave/brave-core/pull/4892/commits/c6cb8d55b95ea8003628191f00065204eeff5447